hosted_control_planes/index.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="hcp-overview"]
+= Hosted control planes overview
+include::_attributes/common-attributes.adoc[]
+:context: hcp-overview
+
+You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With hosted control planes for {product-title}, you create control planes as pods on a hosting cluster without the need for dedicated physical machines for each control plane.
+
+toc::[]
+
+include::modules/hosted-control-planes-overview.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hypershift-addon-intro[HyperShift add-on (Technology Preview)]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)]
+
+
+include::modules/hosted-control-planes-version-support.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about the `HostedCluster` and `NodePool` resources, see the link:https://hypershift-docs.netlify.app/reference/api/[HyperShift API Reference].
```

Comment:
Additional resources are bare links, no intro text. Also please verify this is the link you want to merge.
```suggestion
* link:https://hypershift-docs.netlify.app/reference/api/[HyperShift API Reference].
```